### ISSUE TITLE
[Billing App Invoices Tab] Clicking Send E-mail Always Fails and Logs User Out Automatically

### ISSUE THREAD

**Issue by b-f-chan** [overture-stack/billing#110]
In the billing app, on the Invoices tab, if you click Send E-mail on any invoice:

- Request always fails and an error message is displayed
- After the user closes the error message, if they interact in any way with the app, it automatically logs them out

Existing prod issue, seems to be a result of some changes to the Freshbooks API we were not aware of.

**Comment by caravinci** [comment]
Seems the issue was caused by a mismatch of what the UI was passing, and what was actually expected by Freshbooks.

Being logged out upon the failing request seems to be part of an existing error handling pattern, that comes from the auth part of the billing api. I've tried to improve that as part of this ticket, but as it turns out, the scope of it is a bit larger, and modifying that error handling should be a ticket of its own.
